.changeset/four-lights-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
```

Comment:
@jeddy3 This should be a patch release, right?

```suggestion
"stylelint": patch
```

Reply:
Good catch.